realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpAuthenticationServer.java

```diff
@@ -38,7 +38,7 @@
     public static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
     private static final String ACTION_LOGOUT = "revoke"; // Auth end point for logging out users
     private static final String ACTION_CHANGE_PASSWORD = "password"; // Auth end point for changing passwords
-    private static final String ACTION_LOOKUP_USER_ID = "/users/:provider:/:providerId:"; // Auth end point for looking up user id
+    private static final String ACTION_LOOKUP_USER_ID = "users/:provider:/:providerId:"; // Auth end point for looking up user id
```

Comment:
why removing the leading `/` are you aligning URI?

Reply:
Yes, the first part of the URL is normalized so it always ends with `/`